api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/filter/TenantFilter.java

```diff
@@ -63,14 +63,6 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
             return;
         }
 
-        tenant = uriInfo.getQueryParameters().getFirst(TENANT_QUERY_PARAM_NAME);
```

Comment:
You should also update `MISSING_TENANT_MSG`  and remove `TENANT_QUERY_PARAM_NAME`